RD-6402 Update Secrets Widget documentation (#2524)
cloudify-cosmo/docs.getcloudify.org: content/working_with/manager/using-secrets.md

For more commands, see [secrets command line]({{< relref "cli/orch_cli/secrets.md" >}}).

> ### Creating a secret with value from the {{< param cfy_console_name >}}

Reviewer: ```suggestion
### Creating local secret from the {{< param cfy_console_name >}}
```

![Create Secret]( /images/manager/create_secret_dialog.png )

> ### Creating a secret with Secret Provider from the {{< param cfy_console_name >}}

Reviewer: Maybe to refer to terms convention used in this page better would be to have something like:

```suggestion
### Creating remotely provided secret from the {{< param cfy_console_name >}}
```

1. Go to the [Secrets page]({{< relref "working_with/console/pages/secrets-page.md" >}}).
2. Click **Create** in the [Secret Store Management widget]({{< relref "working_with/console/widgets/secrets.md" >}}).
3. Insert the secret key

> 4. Click on the checkbox "Retrieve the secret value from a secret provider"

Reviewer: What do you think about adding something like (to be updated before committing as I didn't include correct link)?:

```suggestion
4. Click on the checkbox "Retrieve the secret value from a secret provider" (you must have at least one secret provider defined in {{< param product_name >}} to be able to use this option, see [Secret Providers widget](link to Secrets Provider widget) for details)
```

Reviewer: As mentioned in the original comment - link to be updated 🙂

Author: didn't notice that 🙈 https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2524/commits/ef89980c9af79fd8763678a8200af9e46696a842 updated link and one small typo :)

5. Select the secret provider from the dropdown list

> 6. Insert the secret path

Reviewer: How about a bit more precise descripton:
```suggestion
6. Insert the secret's path in selected secret provider
```
?
